test/unit_tests/cli/commands/version_check_test.dart

```diff
@@ -40,126 +40,126 @@ void main() {
 
   group('Version Check', () {
     test('is fine with breaking change and major version change', () {
-      bool versionChangeCheckResult = VersionCheck.versionChangeMatchesChanges(
+      final versionChangeCheckResult = VersionCheck.check(
```

Comment:
The tests could be more fine-grained now probably, also checking for expected version etc.